P5/Source/Specs/lb.xml

```diff
@@ -89,15 +89,15 @@ soi-<lb break="no"/>xante &amp; dixhuytiesme livre.
       starts. The <att>n</att> attribute, if used, indicates the number or other value associated
       with the text between this point and the next <gi>lb</gi> element, typically the sequence
       number of the line within the page, or other appropriate unit. This element is intended to be
-      used for marking actual line breaks on a manuscript or printed page, at the point where they
+      used for marking line beginnings on a manuscript or printed page, at the point where they
```

Comment:
As previous comment, perhaps “… used for marking the beginning of each line on a …”?

Reply:
Thank you. Yes, I agree.